src/equations/ideal_glm_mhd_1d.jl

```diff
@@ -272,6 +272,10 @@ function flux_hllc(u_ll, u_rr, orientation::Integer,
     rho_ll, v1_ll, v2_ll, v3_ll, p_ll, B1_ll, B2_ll, B3_ll = cons2prim(u_ll, equations)
     rho_rr, v1_rr, v2_rr, v3_rr, p_rr, B1_rr, B2_rr, B3_rr = cons2prim(u_rr, equations)
 
+    # Total pressure (eq. (12))
```

Comment:
```suggestion
    # Total pressure, i.e., thermal + magnetic pressures (eq. (12))
```
Maybe just a small point of clarification